### DISCUSSION TITLE
Where is the `!!! warning` syntax documented?

### DISCUSSION THREAD

**Discussion by astrojuanlu** [mkdocs/mkdocs#2553] [Q&A] [answered]
Hi folks! I see `!!! warning` and `!!! note` used in several places on the MkDocs documentation itself, but I don't see where are they documented. I expected to find it here https://www.mkdocs.org/user-guide/writing-your-docs/, along with the Markdown dialect that's used.

**Comment by Andre601** [comment, marked as answer]
This is not a direct mkdocs feature, but part of the Markdown library used called admonition.

Material for MkDocs has docs about it.
https://squidfunk.github.io/mkdocs-material/reference/admonitions/

**Reply by linxianmiao** [reply]
https://facelessuser.github.io/pymdown-extensions/
you can see this lib
